lib/plug/quiet_logger.ex

```diff
@@ -8,9 +8,18 @@ defmodule Plug.QuietLogger do
     %{log: log, path: path}
   end
 
+  def call(%{request_path: path} = conn, %{log: :info, path: paths}) when is_list(paths) do
+    cond do
```

Comment:
I normally prefer an `if` statement instead of a binary `cond`, it feels a bit clearer!

`if path in paths, do: conn, else: Plug.Logger.call(conn, log)`

What do you think?